tests/bluetooth/tester/src/btp_cap.c

```diff
@@ -0,0 +1,1028 @@
+	struct bt_bap_stream *stream;
+	struct unicast_connection *u_conn = tester_bap_unicast_conn_get(bt_conn_index(conn));
+
+	u_stream = tester_bap_unicast_stream_find(u_conn, ase_id);
+	if (u_stream == NULL) {
+		/* Configure a new u_stream */
+		u_stream = tester_bap_unicast_stream_alloc(u_conn);
+		if (u_stream == NULL) {
+			LOG_DBG("No streams available");
+
+			return -ENOMEM;
+		}
+	}
+
+	stream = stream_unicast_to_bap(u_stream);
+	bt_cap_stream_ops_register(&u_stream->audio_stream.cap_stream, stream->ops);
+
+	u_stream->conn_id = bt_conn_index(conn);
+	u_stream->ase_id = ase_id;
+	u_stream->cig_id = cig_id;
+	u_stream->cis_id = cis_id;
+	memcpy(&u_conn->qos, qos, sizeof(*qos));
+	memcpy(&u_stream->codec_cfg, codec_cfg, sizeof(*codec_cfg));
+
+	return 0;
+}
+
+static uint8_t btp_cap_unicast_setup_stream(const void *cmd, uint16_t cmd_len,
+					    void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_cap_unicast_setup_stream_cmd *cp = cmd;
+	struct bt_audio_codec_cfg codec_cfg;
+	struct bt_audio_codec_qos qos;
+	struct bt_conn *conn;
+	const uint8_t *ltv_ptr;
+	int err;
+
+	LOG_DBG("");
+
+	conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, &cp->address);
+	if (!conn) {
+		LOG_ERR("Unknown connection");
+
+		return BTP_STATUS_FAILED;
+	}
+
+	memset(&qos, 0, sizeof(qos));
+	qos.phy = BT_AUDIO_CODEC_QOS_2M;
+	qos.framing = cp->framing;
+	qos.rtn = cp->retransmission_num;
+	qos.sdu = sys_le16_to_cpu(cp->max_sdu);
+	qos.latency = sys_le16_to_cpu(cp->max_transport_latency);
+	qos.interval = sys_get_le24(cp->sdu_interval);
+	qos.pd = sys_get_le24(cp->presentation_delay);
+
+	memset(&codec_cfg, 0, sizeof(codec_cfg));
+	codec_cfg.id = cp->coding_format;
+	codec_cfg.vid = cp->vid;
+	codec_cfg.cid = cp->cid;
+
+	ltv_ptr = cp->ltvs;
+	if (cp->cc_ltvs_len != 0) {
+		codec_cfg.data_len = cp->cc_ltvs_len;
+		memcpy(codec_cfg.data, ltv_ptr, cp->cc_ltvs_len);
+		ltv_ptr += cp->cc_ltvs_len;
+	}
+
+	if (cp->metadata_ltvs_len != 0) {
+		codec_cfg.meta_len = cp->metadata_ltvs_len;
+		memcpy(codec_cfg.meta, ltv_ptr, cp->metadata_ltvs_len);
+	}
+
+	err = cap_unicast_setup_stream(conn, cp->ase_id, cp->cis_id, cp->cig_id, &codec_cfg, &qos);
+	bt_conn_unref(conn);
+	if (err != 0) {
+		return BTP_STATUS_FAILED;
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_cap_unicast_audio_start(const void *cmd, uint16_t cmd_len,
+					   void *rsp, uint16_t *rsp_len)
+{
+	int err;
+	size_t stream_count = 0;
+	struct unicast_group *u_group;
+	const struct btp_cap_unicast_audio_start_cmd *cp = cmd;
+	struct bt_cap_unicast_audio_start_param start_param;
+	struct bt_cap_unicast_audio_start_stream_param stream_params[ARRAY_SIZE(set_members) *
+								     MAX_STREAMS_COUNT];
+
+	LOG_DBG("");
+
+	err = tester_bap_client_create_unicast_group(cp->cig_id, &u_group);
+	if (err != 0) {
+		LOG_ERR("Failed to create unicast group");
+
+		return BTP_STATUS_FAILED;
+	}
+
+	for (size_t conn_index = 0; conn_index < (size_t)ARRAY_SIZE(set_members); conn_index++) {
```

Comment:
```suggestion
	for (size_t conn_index = 0; conn_index < ARRAY_SIZE(set_members); conn_index++) {
```

`ARRAY_SIZE` already returns a `size_t`